### PR TITLE
docs: add Yeecord to showcase

### PR DESCRIPTION
**Pull request by Gary50613**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs

No

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -2269,6 +2269,14 @@ const Users: User[] = [
     source: 'https://github.com/woodpecker-ci/woodpecker/tree/master/docs',
     tags: ['opensource'],
   },
+  {
+    title: 'YEE式機器龍 (Yeecord)',
+    description: '一隻全中文的機器人，並結合了音樂系統、RPG、動態語音頻道、跨群聊天等功能，讓Discord不再只是聊天軟體。',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:2274]: Please use English for the title & description.

**Comment by Gary50613** [comment]
hmm I just signed the CLA, should I wait for the bot to mark it as passed?

**Comment by Josh-Cena** [comment]
Yes, you may need to wait for up to a few hours. Not a huge problem.

**Comment by Gary50613** [comment]
ok I'll wait thanks for help reviewing this PR! tysm

**Review by Josh-Cena — APPROVED** [review]
That's very cute. Thanks!

**Pull request merged**